src/schema/message.js

```diff
@@ -11,7 +11,32 @@ export const messageSchema = new Schema({
     code_block: schema.spec.nodes.get('code_block'),
     text: schema.spec.nodes.get('text'),
     hard_break: schema.spec.nodes.get('hard_break'),
-    image: schema.spec.nodes.get('image'),
+    image: {
+      ...schema.spec.nodes.get('image'),
+      attrs: {
+        ...schema.spec.nodes.get('image').attrs,
+        height: {default: null}
+      },
+      parseDOM: [{
+        tag: 'img[src]',
+        getAttrs: dom => ({
+          src: dom.getAttribute('src'),
+          title: dom.getAttribute('title'),
+          alt: dom.getAttribute('alt'),
+          height: parseInt(dom.style.height, 10)
```

Comment:
```suggestion
          height: parseInt(dom.style.height)
```

The default is base-10 anyway, no need for this

Reply:
@scmmishra Showing some eslint warning. That's the reason I added it.
<img width="534" alt="image" src="https://github.com/chatwoot/prosemirror-schema/assets/64252451/bee8788a-70cc-4547-8349-fc08fdebb1ba">

Reply:
Done